src/pages/BanquetFacility/BanquetFacility.js

```diff
@@ -1,51 +1,37 @@
-/*****************************
-  TODO: Refactor to use builder data instead
-  ###
-  First console.log(data); to see how the data is structured
-  Our new data is going to live under 'builder'
-  Second lets componentize these sections now, we should end up with 3 components
-    1. Image Banner
-    2. Title Subtitle Body
-    3. Image Title Body Button
-  NOTE: These components should be placed in folder name with leading name, exampled Title Subtitle Body should live in a title directory
-  Lastly, please delete this comment
-******************************/
-
 import React from 'react';
 import Layout from 'components/Layout/Layout.js';
 import Container from 'components/Container/Container.js';
-import Button from 'components/Button/Button.js';
-import Image from 'components/Image/Image.js';
-
-function createMarkup(body) {
-  return {__html: body};
-  }
+import ImageBanner from 'components/Image/ImageBanner/ImageBanner.js';
+import ImageTitleBodyButton from 'components/Image/ImageTitleBodyButton/ImageTitleBodyButton.js';
+import TitleSubtitleBody from 'components/Title/TitleSubtitleBody/TitleSubtitleBody.js';
 
-const BanquetFacility = ({data}) => {
-  if (!data) return null;
+const BanquetFacility = ({ data }) => {
+if (!data) return null;
+let { builder } = data;
+let [image_banner, title_subtitle_body, image_title_body_button] = builder;
```

Comment:
🤔 soo this works fine now, but this is pretty brittle, we cant guarantee order but maybe thats something to think about later...